packages/web-twig/src/Resources/components/CheckboxField/stories/CheckboxFieldDisabled.twig

```diff
@@ -1,13 +1,16 @@
 <CheckboxField
-    id="checkboxfield3"
+    id="checkboxfieldDisabled"
     isDisabled
-    label="Checkbox unselected"
-    name="checkboxfield3"
+    label="Checkbox Label"
+    name="checkboxfieldDisabled"
 />
+
 <CheckboxField
-    id="checkboxfield7"
+    helperText="Helper text"
+    id="checkboxfieldDisabledHelperText"
     isChecked
     isDisabled
-    label="Checkbox selected"
-    name="checkboxfield7"
+    isRequired
+    label="Checkbox Label"
+    name="checkboxfieldDisabledHelperText"
 />
```

Comment:
Please add a demo of the indeterminate state like we have in the `web` package:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/ed0412d3-7d26-4778-8d84-04cc47890daf)